test/coverlet.core.tests/Helpers/InstrumentationHelperTests.cs

```diff
@@ -3,14 +3,15 @@
 using Xunit;
 using System.Collections.Generic;
 using System.Linq;
+using Castle.Core.Internal;
 using Moq;
 using Coverlet.Core.Abstractions;
 
 namespace Coverlet.Core.Helpers.Tests
 {
     public class InstrumentationHelperTests
     {
-        private readonly InstrumentationHelper _instrumentationHelper =
+        private InstrumentationHelper _instrumentationHelper =
```

Comment:
I prefer keep this `readonly` and define new one only for that test wdym?
Like you did in `CanInstrumentFSharpAssemblyWithAnonymousRecord`

Reply:
Done